src/main/java/uk/gov/mca/beacons/service/model/BeaconUse.java

```diff
@@ -86,7 +86,8 @@
 
   private String rigPlatformLocation;
 
-  private boolean mainUse;
+  @NotNull
+  private Boolean mainUse;
```

Comment:
@tm1303 As discussed, require the client to set the main use on initial registration